File: testdata/bench/php/BenchmarkTest00020.php
<?php
$pdo = new PDO("mysql:host=localhost;dbname=test", "root", "");
$email = $_POST['email'];
$stmt = $pdo->prepare("SELECT * FROM users WHERE email = ?");
$stmt->execute([$email]);
$user = $stmt->fetch(PDO::FETCH_ASSOC);
?>

File: testdata/bench/perl/BenchmarkTest00020.pl
use Dancer2;
post '/update' => sub {
    my $id = params->{id};
    my $status = params->{status};
    my $dbh = DBI->connect("dbi:SQLite:dbname=tasks.db");
    my $sth = $dbh->prepare("UPDATE tasks SET status = ? WHERE id = ?");
    $sth->execute($status, $id);
    return { ok => 1 };
};
